helm/templates/metric-anomaly-task-manager/deployment.yaml

```diff
@@ -65,7 +65,7 @@ spec:
             - name: LOG4J_CONFIGURATION_FILE
               value: "/var/{{ .Chart.Name }}/log/log4j2.properties"
             - name: JAVA_TOOL_OPTIONS
```

Comment:
JAVA_TOOL_OPTIONS shouldn't be used, use JAVA_OPTS instead https://github.com/hypertrace/config-service/blob/777df107b0141eb3368516c6527b4b846ea2d539/helm/templates/deployment.yaml#L65-L66